state/pg_queries.go

```diff
@@ -309,3 +321,10 @@ const GetWorkerSQL = WorkerSelect + "\nwhere worker_type = $1 and engine = $2"
 // worker type; locks the row.
 //
 const GetWorkerSQLForUpdate = GetWorkerSQL + " for update"
+
+const taskTypeSelect = `
+  SELECT id, alias, schema::TEXT, template
+  FROM task_type
+`
+const ListTaskTypeSQL = taskTypeSelect + "\n limit $1 offset $2"
```

Comment:
Orthogonally ties into versioning, how would multiple versions of the same task? get latest or last version?